rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -605,10 +605,13 @@ void gs_frame::close()
 		if (!Emu.IsStopped())
 		{
 			// Blocking shutdown request. Obsolete, but I'm keeping it here as last resort.
-			Emu.GracefulShutdown(true, false);
+			Emu.after_kill_callback = [this](){ deleteLater(); };
```

Comment:
why keep the object? The window is already hidden at this point.

Reply:
I didnt change anything here, it should behave the same as before.

Reply:
then why did you change it at all?

Reply:
Kill is now async, this is just an adjustment for it.